analyticConfigurations/metric_meta-consul.json

```diff
@@ -0,0 +1,49 @@
+{
```

Comment:
Can you use the Metricly CLI to format this package?

Reply:
That is very handy. Will do from now on.